commercial/app/views/hosted/guardianHostedVideo.scala.html

```diff
@@ -5,6 +5,64 @@
 @import views.html.hosted.guardianHostedCta
 
 @main(page, Some("commercial")) { } {
+    <!--[if (gt IE 9)|(IEMobile)]><!-->
+    <style>
+    .hosted-tone {
+        color: @{page.campaign.brandColour};
+    }
+
+    .hosted-tone-bg {
+        background-color: @{page.campaign.brandColour};
+    }
+
+    .hosted-tone-btn,
+    .hosted-tone-btn:focus,
+    .hosted-tone-btn:hover {
+        background-color: @{page.campaign.brandColour};
+        border-color: @{page.campaign.brandColour};
+    }
+
+    .hosted-page .hosted-next-autoplay .hosted-next-autoplay__poster-timer,
+    .hosted-page .hosted-next-autoplay .hosted-next-autoplay__tile {
+        background-color: @{page.campaign.brandColour};
+    }
+
+    .hosted-page .hosted-next-autoplay .hosted-next-autoplay__video,
```

Comment:
There's some pretty deeply nested classes in this CSS - It seems if they're all prefixed with `.hosted-` you should be able to just use the last class? If you're overriding specificity, I'd try and revisit the classes you're overriding to see if you can reduce theirs.